git_machete/tests/functional/test_machete.py

```diff
@@ -87,29 +95,29 @@ def launch_command(*args: str) -> str:
                 git.flush_caches()
             return out.getvalue()
 
-    def assert_command(self, cmd: Iterable[str], expected_result: str) -> None:
-        self.assertEqual(self.launch_command(*cmd), self.adapt(expected_result))
+    def assert_command(self, cmds: Iterable[str], expected_result: str) -> None:
+        self.assertEqual(self.launch_command(*cmds), self.adapt(expected_result))
 
     def setUp(self) -> None:
         # Status diffs can be quite large, default to ~256 lines of diff context
         # https://docs.python.org/3/library/unittest.html#unittest.TestCase.maxDiff
         self.maxDiff = 80 * 256
 
-        self.setup = SandboxSetup()
+        self.repo_sandbox = GitRepositorySandbox()
 
         (
-            self.setup
+            self.repo_sandbox
             # Create the remote and sandbox repos, chdir into sandbox repo
-            .new_repo(self.setup.remote_path, "--bare")
-            .new_repo(self.setup.sandbox_path)
-            .execute(f"git remote add origin {self.setup.remote_path}")
+            .new_repo(self.repo_sandbox.remote_path, "--bare", "-q")
```

Comment:
Hmmm looking at the impl of `new_repo`... isn't `-q` ignored here? 🤔

Reply:
But generally good idea, I'd add some `--quiet` option wherever possible to the setup commands (as opposed to the actual tested `git machete` commands) wherever possible to cut on spam in output

Reply:
You're right, it was ignored. When I swapped the order, so the -q comes before --bare it hides hints about "main/master" naming convention.